cmd/minikube/cmd/start.go

```diff
@@ -175,7 +173,7 @@ var startCmd = &cobra.Command{
 	Use:   "start",
 	Short: "Starts a local kubernetes cluster",
 	Long: `Starts a local kubernetes cluster using VM. This command
-assumes you have already installed one of the VM drivers: virtualbox/parallels/vmwarefusion/kvm/xhyve/hyperv.`,
+assumes you have already installed one of the VM drivers: virtualbox/parallels/vmwarefusion/kvm/hyperv.`,
```

Comment:
kvm2 ?

Reply:
Maybe something like: `fmt.Sprintf("....%v", constants.SupportedVMDrivers))`. Then once the PR https://github.com/kubernetes/minikube/pull/4738 is merged, it would list the correct drivers for each platform. 

wdyt?

Reply:
I didn't find the long help here particularly useful, so I removed it.